packages/directory/addon/components/dir-icon.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * {{dir-icon
+ *    value=value
+ * }}
+ */
+import Component from '@ember/component';
+import layout from '../templates/components/dir-icon';
+import { get, getWithDefault, computed } from '@ember/object';
+import fileTypes from 'navi-directory/utils/enums/file-types';
+
+export default Component.extend({
```

Comment:
Didn't we discuss moving this out of this PR, since there was a plan for combining this with the name to create a new cell component?